src/components/ArticleVotes.jsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { patchVotesByArticleId } from "../api";
+
+function ArticleVotes({ article, currentVotes }) {
+    
+  const [votes, setVotes] = useState(0);
+  const [error, setError] = useState(null);
+  const [optimisticMsg, setOptimisitcMsg] = useState(null);
+
+  const article_id = article.article_id;
+
+  const handleVote = (vote) => {
+    setVotes(votes + vote);
+    patchVotesByArticleId(article_id, vote)
+      .then(() => {
+        setOptimisitcMsg("Vote Submitted!");
+      })
+      .catch((error) => {
+        setError("Unable to update votes. Please try again.");
+      });
+  };
+
+  return (
+    <div id="article-vote">
+      <button className="vote-button" disabled={votes === 1} onClick={() => handleVote(1)}>
+        +
+      </button>
+      {article.votes + votes}
+      <button className="vote-button" disabled={votes === -1} onClick={() => handleVote(-1)}>
```

Comment:
Great use of the `disabled` property to make sure the user can't use the button when it's already been pressed